lib/constants.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  COMPONENT_OPTIONS_BLOCK: 'script',
+  COMPONENT_OPTIONS_KEY: 'sitemap'
```

Comment:
To be customisable, you can set that key in the sitemap configuration in `nuxt.config.js` instead of create a constant.